### PR TITLE
Adds tool to set up save files.

### PR DESCRIPTION
**Pull request by MrTyton**
Must be run from the main directory that the TAS has been cloned to.

Someone needs to modify the setup instructions.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+absl-py=1.2.0
```
Comment (jlward) [inline requirements.txt:1]: Might be worth dumping this until `dev-requirements.txt` since it's only needed if you are working on the code

**Review by jlward — APPROVED** [review]
One minor nitpick, but otherwise looks good

File: requirements.txt (hunk 1 above)
Comment (MrTyton) [reply requirements.txt:1]: Ok

Comment (MrTyton) [reply requirements.txt:1]: Wait, do we have one already set up and I missed it?

Comment (MrTyton) [reply requirements.txt:1]: NHope for some reason git isn't seeing the dev-requirements file so letting someone else do this.

**Pull request merged**